app/assets/stylesheets/custom.css.scss

```diff
@@ -8,16 +20,28 @@ body {
 .header {
   padding-left: 15px;
   padding-right: 15px;
+  padding-bottom: 19px;
   margin-bottom: 15px;
   border-bottom: 1px solid #e5e5e5;
+  h3 {
+    padding-bottom: 0px;

```

Comment:
Properties should be ordered line-height, margin-bottom, margin-top, padding-bottom<br>`0px` should be written without units as `0`